kroxylicious-api/src/main/java/io/kroxylicious/proxy/service/BaseContributor.java

```diff
@@ -178,7 +189,7 @@ public BaseContributorBuilder<L, D> add(String typeName, Supplier<L> instanceFun
          * @return this
          */
         public BaseContributorBuilder<L, D> add(String typeName, Function<D, L> instanceFunction) {
-            return add(typeName, BaseConfig.class, (context, config) -> instanceFunction.apply(context));
+            return add(typeName, BaseConfig.class, (context, config) -> instanceFunction.apply(context), true);
```

Comment:
Surely configurationRequired should be false in this case?

Reply:
Yes your right. 

I've added an additional test in 3630b049077878ff557ba343fc1094bc5b41fae4 I may go back and remove the BaseConfig case. Once I've cleared up in my head what the config loading will do in practice.